mars/scheduler/tests/integrated/base.py

```diff
@@ -195,11 +195,14 @@ def start_processes(self, n_schedulers=2, n_workers=2, etcd=False, cuda=False, m
         check_time = time.time()
         while True:
             try:
-                started_schedulers = self.cluster_info.get_schedulers()
-                if len(started_schedulers) < n_schedulers:
-                    raise ProcessRequirementUnmetError('Schedulers does not met requirement: %d < %d.' % (
-                        len(started_schedulers), n_schedulers
-                    ))
+                try:
+                    started_schedulers = self.cluster_info.get_schedulers()
+                    if len(started_schedulers) < n_schedulers:
+                        raise ProcessRequirementUnmetError('Schedulers does not met requirement: %d < %d.' % (
+                            len(started_schedulers), n_schedulers
+                        ))
+                except Exception as e:
```

Comment:
IMO it's better to catch the line of `started_schedulers = self.cluster_info.get_schedulers()` only.